pavelib/assets.py

```diff
@@ -109,6 +109,28 @@ def on_modified(self, event):
             traceback.print_exc()
 
 
+class XModuleAssetsWatcher(PatternMatchingEventHandler):
+    """
+    Watches for css and js file changes
+    """
+    ignore_directories = True
+    patterns = ['*.css', '*.js']
+
+    def register(self, observer):
+        """
+        Register files with observer
+        """
+        observer.schedule(self, 'common/lib/xmodule/', recursive=True)
+
+    def on_modified(self, event):
+        print('\tCHANGED:', event.src_path)
+        try:
+            process_xmodule_assets()
+            restart_django_servers()  # To refresh the hash values of static xmodule content
+        except Exception:  # pylint: disable=broad-except
```

Comment:
What can be done to follow pylint's suggestion here? What kind of exception would we normally get? I agree it would be good to avoid broad excepts when possible.